src/components/HomePage.jsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import { useEffect, useState } from 'react';
+import { fetchArticles } from '../ApiRequests';
+import './HomePage.css';
+import { Link } from "react-router-dom";
+
+
+function Homepage() {
+    const [articles, setArticles] = useState([]);
+    useEffect(() => {
+      fetchArticles().then((articlesFromApi) => {
+      setArticles(articlesFromApi);
+    });
+    }, []);
+  
+  return articles.length ? (
```

Comment:
No completely sure what this ternary is achieving. What happens if its not there and it just a return?